sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/utils/CosmosPatchTestHelper.scala

```diff
@@ -171,4 +183,13 @@ object CosmosPatchTestHelper {
   columnConfigString += s"op(${columnConfig.operationType})"
   columnConfigString
  }
+
+ /***
+  * Get partition key path without "/" at the beginning
+  * @param partitionKeyDefinition the partition key definition
+  * @return the partition key path without
+  */
+ def getStrippedPartitionKeyPath(partitionKeyDefinition: PartitionKeyDefinition): String = {
```

Comment:
This wouldn't work with hierarchical partitioning - not a blocker for me because hierarchical partitioning will cause problems in few other places and is not officially supported yet - but please add a todo so we can identify it when going through the backlog - no GitHub Item strictly needed just TODO comment for now should be fine.

Reply:
Thanks, added the following comments:
  // TODO: Reexamine the logic here when hierarchical partitioning being supported